### PR TITLE
Update Spectrum Scale ECE profile

### PR DESCRIPTION
**Pull request by bolinches**
Signed-off-by: Luis Bolinches <luis.bolinches@fi.ibm.com>

### CODE REVIEW

File: profiles/spectrumscale-ece/tuned.conf [hunk 1]
```diff
@@ -6,7 +6,29 @@
...
+
+[disk-sas]
+type=disk
+devices = sd*
+elevator = mq-deadline
+readahead = 0
+
+[disk-nvme]
+type=disk
+devices = nvme*
```
Comment (olysonek) [inline profiles/spectrumscale-ece/tuned.conf:32]: Does the tuning really get applied to these disks? I remember there was a problem with Tuned not recognizing nvme disks.

Comment (bolinches) [reply profiles/spectrumscale-ece/tuned.conf:32]: Well on RH8 tests it does at least in our lab

Comment (olysonek) [reply profiles/spectrumscale-ece/tuned.conf:32]: OK. I just wanted to ask.

Comment (bolinches) [reply profiles/spectrumscale-ece/tuned.conf:32]: Sure. We have not tested on Fedora but the goal of this profile is not to be run in Fedora as ECE itself is not supported on Fedora. We been running with this parameters for over a month on many test clusters and as soon we have this published we will push for the RH move.

**Comment by bolinches** [comment]
@olysonek Hi I checked tuned-profiles-spectrumscale-2.14.0-2.fc32.noarch.rpm that was build after this PR was accepted and I do not see this is the changelog. Is it coming on a later build? Thanks

**Comment by olysonek** [comment]
@yarda has to merge it first.

**Comment by yarda** [comment]
Thanks, merging.

**Comment by bolinches** [comment]
Hi @yarda I might be missing something. But I still do not see the changes. I tested on F32 and rawhide (F34)

tuned-profiles-spectrumscale-2.14.0-2.fc32.noarch and tuned-profiles-spectrumscale-2.14.0-3.fc33.noarch respectively (not sure why FC33 package tag on rawhide)

The content of /usr/lib/tuned/spectrumscale-ece/tuned.conf on both of those is

#
# tuned configuration
#

[main]
summary=Optimized for Spectrum Scale Erasure Code Edition Servers
include=throughput-performance

[sysctl]
net.ipv4.tcp_window_scaling = 1
net.ipv4.tcp_timestamps = 1
kernel.numa_balancing = 1

Which does not include this merge. What I am doing wrong here? I am trying to get this sorted so I can move to open a request to include on RHEL ... 

Thanks

**Comment by yarda** [comment]
Hi @bolinches the changes are merged upstream and it will go to 2.15.0 release which is preliminary planned to autumn/winter. If you want the changes to appear in Fedora sooner, please open Fedora bug on component Tuned, we could either backport the patch to Fedora or make e.g. minor 2.14.1 upstream release (in such case we don't need the Fedora bug).  Regarding RHEL-8 there is https://bugzilla.redhat.com/show_bug.cgi?id=1874052 which will cover it.

**Comment by bolinches** [comment]
Thanks a lot

**Pull request merged**